### ISSUE TITLE
After upgrading from 1.1.15 to 1.2.3, the node crashed

### ISSUE THREAD

**Issue by WangXiaopai595** [node-real/bsc-erigon#352]
Erigon version: `1.2.3`

Command:
```
erigon --txpool.disable --maxpeers=1000 --p2p.protocol=66,67 --metrics.addr=0.0.0.0 --log.console.verbosity=info --db.pagesize=16k --db.size.limit=10T --datadir ./data --private.api.addr=localhost:9090 --chain=bsc --metrics --log.dir.path ./log --nat=any --snap.stop
```

Error message：
```
[INFO] [04-07|15:41:33.869] logging to file system                   log dir=./log file prefix=erigon log level=info json=false
[INFO] [04-07|15:41:33.869] Enabling metrics export to prometheus    path=http://0.0.0.0:6060/debug/metrics/prometheus
[INFO] [04-07|15:41:33.869] Build info                               git_branch= git_tag= git_commit=
[INFO] [04-07|15:41:33.869] Starting Erigon on BSC mainnet... 
[INFO] [04-07|15:41:33.874] Maximum peer count                       ETH=1000 total=1000
[INFO] [04-07|15:41:33.874] starting HTTP APIs                       port=8545 APIs=eth,erigon,engine
[INFO] [04-07|15:41:33.874] torrent verbosity                        level=WRN
[INFO] [04-07|15:41:35.883] Set global gas cap                       cap=50000000
[INFO] [04-07|15:41:35.895] [Downloader] Running with                ipv6-enabled=true ipv4-enabled=true download.rate=16mb upload.rate=4mb
[INFO] [04-07|15:41:35.895] Opening Database                         label=chaindata path=/home/erigon/data/chaindata
[INFO] [04-07|15:41:35.903] [db] open                                lable=chaindata sizeLimit=10TB pageSize=16384
[INFO] [04-07|15:41:35.905] Initialised chain configuration          config="{ChainID: 56 Ramanujan: 0, Niels: 0, MirrorSync: 5184000, Bruno: 13082000, Euler: 18907621, Gibbs: 23846001, Nano: 21962149, Moran: 22107423, Planck: 27281024, Luban: 29020050, Plato: 30720096, Hertz: 31302048, Hertzfix: 34140700, ShanghaiTime: 1705996800, KeplerTime 1705996800, FeynmanTime 1713419340, FeynmanFixTime 1713419340, Engine: parlia}" genesis=0x0d21840abff46b96c84b2ac9e10e4f5cdaeb5693cb665db62a2f3b02d2d57b5b
[DBUG] [04-07|15:41:40.263] [db] open                                lable=downloader sizeLimit=16GB pageSize=4096
[DBUG] [04-07|15:41:45.548] [snapshots] webseed providers            http=0 disk=0
[INFO] [04-07|15:41:45.549] Initialising Ethereum protocol           network=56
[INFO] [04-07|15:41:45.549] Opening Database                         label=parlia path=/home/erigon/data/parlia
[DBUG] [04-07|15:41:45.555] [db] open                                lable=consensus sizeLimit=2TB pageSize=4096
[INFO] [04-07|15:41:46.100] Starting private RPC server              on=localhost:9090
[INFO] [04-07|15:41:46.100] new subscription to logs established 
[INFO] [04-07|15:41:46.101] rpc filters: subscribing to Erigon events 
[DBUG] [04-07|15:41:46.101] Establishing event subscription channel with the RPC daemon ... 
[INFO] [04-07|15:41:46.101] new subscription to newHeaders established 
[INFO] [04-07|15:41:46.103] Reading JWT secret                       path=/home/erigon/data/jwt.hex
[DBUG] [04-07|15:41:46.105] [db] open                                lable=sentry sizeLimit=8GB pageSize=4096
SIGSEGV: segmentation violation
PC=0x723a1ddff73c m=47 sigcode=1
signal arrived during cgo execution

goroutine 509 [syscall]:
runtime.cgocall(0x23d11d0, 0xc0caf0ed88)
	/usr/local/go/src/runtime/cgocall.go:157 +0x4b fp=0xc0caf0ed60 sp=0xc0caf0ed28 pc=0x42648b
net._C2func_getaddrinfo(0xc0629f6130, 0x0, 0xc0a0c1e420, 0xc09701c138)
	_cgo_gotypes.go:100 +0x55 fp=0xc0caf0ed88 sp=0xc0caf0ed60 pc=0x623d95
net._C_getaddrinfo.func1(0x42f5e5?, 0x8?, 0x2675e60?, 0x1?)
	/usr/local/go/src/net/cgo_unix_cgo.go:78 +0x7a fp=0xc0caf0edf0 sp=0xc0caf0ed88 pc=0x62471a
net._C_getaddrinfo(0xc001e5e07e?, 0x9?, 0x0?, 0x0?)
	/usr/local/go/src/net/cgo_unix_cgo.go:78 +0x13 fp=0xc0caf0ee20 sp=0xc0caf0edf0 pc=0x624653
net.cgoLookupHostIP({0xc001e5e078, 0x3}, {0xc001e5e07e, 0x9})
	/usr/local/go/src/net/cgo_unix.go:166 +0x24f fp=0xc0caf0ef60 sp=0xc0caf0ee20 pc=0x5eed4f
net.cgoLookupIP.func1()
	/usr/local/go/src/net/cgo_unix.go:215 +0x25 fp=0xc0caf0ef90 sp=0xc0caf0ef60 pc=0x5ef465
net.doBlockingWithCtx[...].func1()
	/usr/local/go/src/net/cgo_unix.go:56 +0x35 fp=0xc0caf0efe0 sp=0xc0caf0ef90 pc=0x624af5
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0caf0efe8 sp=0xc0caf0efe0 pc=0x490f41
created by net.doBlockingWithCtx[...] in goroutine 508
	/usr/local/go/src/net/cgo_unix.go:54 +0xd8

goroutine 1 [sleep]:
runtime.gopark(0x2ef845b684913e?, 0x39a0290?, 0x0?, 0x0?, 0xc058f22b10?)
	/usr/local/go/src/runtime/proc.go:398 +0xce fp=0xc00363b490 sp=0xc00363b470 pc=0x45d52e
time.Sleep(0x989680)
	/usr/local/go/src/runtime/time.go:195 +0x125 fp=0xc00363b4d0 sp=0xc00363b490 pc=0x48df05
github.com/ledgerwatch/erigon/eth.(*Ethereum).Start(0xc0001e2b00)
	/go/src/github.com/node-real/erigon/eth/backend.go:1384 +0x4f fp=0xc00363b678 sp=0xc00363b4d0 pc=0x23aefef
github.com/ledgerwatch/erigon/node.(*Node).Start(0xc0019a20e0)
	/go/src/github.com/node-real/erigon/node/node.go:124 +0x2c8 fp=0xc00363b778 sp=0xc00363b678 pc=0x2107428
github.com/ledgerwatch/erigon/node.StartNode(0xc0019a20e0)
	/go/src/github.com/node-real/erigon/node/node.go:413 +0x1c fp=0xc00363b7d0 sp=0xc00363b778 pc=0x210951c
github.com/ledgerwatch/erigon/turbo/node.(*ErigonNode).run(...)
	/go/src/github.com/node-real/erigon/turbo/node/node.go:52
github.com/ledgerwatch/erigon/turbo/node.(*ErigonNode).Serve(0xc06ce65bd0)
	/go/src/github.com/node-real/erigon/turbo/node/node.go:32 +0x5b fp=0xc00363b820 sp=0xc00363b7d0 pc=0x23b379b
main.runErigon(0xc002934b80)
	/go/src/github.com/node-real/erigon/cmd/erigon/main.go:75 +0x405 fp=0xc00363ba90 sp=0xc00363b820 pc=0x23ceba5
github.com/ledgerwatch/erigon/turbo/app.MakeApp.func1(0xc002934b80)
	/go/src/github.com/node-real/erigon/turbo/app/make_app.go:52 +0x19f fp=0xc00363bbc0 sp=0xc00363ba90 pc=0x23c225f
github.com/urfave/cli/v2.(*Command).Run(0xc0002758c0, 0xc002934b80, {0xc0001cc000, 0xf, 0xf})
	/root/go/pkg/mod/github.com/urfave/cli/v2@v2.27.1/command.go:279 +0x9dd fp=0xc00363be50 sp=0xc00363bbc0 pc=0x7d351d
github.com/urfave/cli/v2.(*App).RunContext(0xc001419c00, {0x397da60?, 0x5d19dc0}, {0xc0001cc000, 0xf, 0xf})
	/root/go/pkg/mod/github.com/urfave/cli/v2@v2.27.1/app.go:337 +0x5db fp=0xc00363beb8 sp=0xc00363be50 pc=0x7cfe5b
github.com/urfave/cli/v2.(*App).Run(...)
	/root/go/pkg/mod/github.com/urfave/cli/v2@v2.27.1/app.go:311
main.main()
	/go/src/github.com/node-real/erigon/cmd/erigon/main.go:34 +0x85 fp=0xc00363bf40 sp=0xc00363beb8 pc=0x23ce6c5
runtime.main()
	/usr/local/go/src/runtime/proc.go:267 +0x2bb fp=0xc00363bfe0 sp=0xc00363bf40 pc=0x45d0bb
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc00363bfe8 sp=0xc00363bfe0 pc=0x490f41
......
```

**Comment by WangXiaopai595** [comment]
Unable to fix restart, using 1.1.17 prompt: `[EROR] [04-07 | 16:52:08.604] Erigon startup err=" migrator. VerifyVersion: cannot downgrade minor DB version from 6.1 to 6.0 "`
 using version 1.2 *, crashes directly

**Comment by WangXiaopai595** [comment]
The problem has been successfully resolved, thank you very much
